Production/crab/crab_cmd.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
Move crab_cmd.py, crab_submit.py and crab_submit_file.py into `Production/scripts` directory. In this way, it is possible to run them from any directory without specifying the full path, e.g. `crab_cmd.py --workArea work-area --cmd status`

Reply:
I did it but it is still not quite use-friendly because we still have to write full path to the Production.py (it is in python folder) or to the config file

Reply:
Writing full path to Production.py is unavoidable, as far as I see. However, you can use `$CMSSW_BASE/src/TauMLTools/Production/python/Production.py`, so the command line will stay the same independently in which directory you are...